kickstart: Fix gui with --useexisting in ks file

Anaconda will erase all the partitioning information when using the
graphical mode because it runs the cleardiskssel screen.  It should
skip this when ever the parition data is specified by a kick start
file.

---
 pyanaconda/kickstart.py |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -486,6 +486,9 @@ class LogVolData(commands.logvol.F15_Log
                 if nformat:
                     dev.format = nformat
                     dev.format.exists = True
+            # Skip the UI screen for cleardiskssel
+            # else all the lv data will erased
+            self.anaconda.dispatch.skip_steps("cleardiskssel")
             self.anaconda.dispatch.skip_steps("partition", "parttype")
             return
 
@@ -807,6 +810,9 @@ class PartitionData(commands.partition.F
                 if nformat:
                     dev.format = nformat
                     dev.format.exists = True
+            # Skip the UI screen for cleardiskssel
+            # else all the partition data will erased
+            self.anaconda.dispatch.skip_steps("cleardiskssel")
             self.anaconda.dispatch.skip_steps("partition", "parttype")
             return
 
